experimental/sql/src/impl_to_iterators.py

```diff
@@ -23,43 +23,49 @@
 # `Rewriter`s. All other `Rewriter`s inherit from that class.
 
 
+def convert_datatype(type_: RelImpl.DataType) -> Attribute:
```

Comment:
I decided to move these functions out in order to use them in `get_batch_and_name_list`. While I had an earlier version where this was done using a Rewriter, this version was not very clean IMO, as it matched on function ops, so it would also match on the functions added for map and filter.